deploy/nexodus/overlays/prod/kustomization.yaml

```
@@ -4,58 +4,58 @@ resources:
   - ../../base
   - issuer.yaml
   - serviceaccount.yaml
-namespace: apex
+namespace: nexodus
 configMapGenerator:
   - name: auth-config
     behavior: replace
     literals:
-      - hostname=auth.apex-hosted.cloud
-      - frontend-url=https://apex-hosted.cloud
+      - hostname=auth.nexodus.io
+      - frontend-url=https://nexodus.io
```

I'd like to use a subdomain so we can use `nexodus.io` to reach the project's home page

```suggestion
      - frontend-url=https://try.nexodus.io
```